react/week4/github_users_search/src/UserList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import searchContext from "./context";
+import './index.css';
+import UserRepo from "./UserRepo";
+
+function UserList() {
+    const { usersArray, isClickedUser, handleClickSearch} = React.useContext(searchContext);
+
+    return (
+        <div className='lists'>
+            {console.log(isClickedUser)}
```

Comment:
Could you remove the `console.log` call? This is useful for debug, but it should not be part of the final code.